pkg/storage/sql/sql.go

```diff
@@ -0,0 +1,87 @@
+package sql
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"github.com/trustwallet/blockatlas/pkg/storage/util"
+)
+
+type sql struct {
+	Client *gorm.DB
+}
+
+type Handler func(db *gorm.DB, value interface{}) error
```

Comment:
can you also add methods to add indexes?

Reply:
whats do you mean?

we can add an indexer in gorm using `gorm:"index:addr"` inside the struct

Reply:
👍